keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,702 @@
+    },
+    "optionsConnectedDatabasesNotFound": {
+        "message": "No connected databases found.",
+        "description": "Shown in the connected databases table when there is no connected databases."
+    },
+    "optionsDatabasesRemoveIdentifier": {
+        "message": "Remove identifier from database list?",
+        "description": "Confirmation text when removing database from the list."
+    },
+    "optionsDatabasesRemoveIdentifierConfirmFirst": {
+        "message": "Do you really want to remove the following identifier from the database list?",
+        "description": "Confirmation text when removing database from the list."
+    },
+    "optionsDatabasesRemoveIdentifierConfirmSecond": {
+        "message": "You can reconnect your database at any time.",
+        "description": "Confirmation text when removing database from the list, second part."
+    },
+    "optionsDatabaseIdentifier": {
+        "message": "Identifier",
+        "description": "Database list column title."
+    },
+    "optionsDatabaseKey": {
+        "message": "Key",
+        "description": "Database list column title."
+    },
+    "optionsDatabaseLastUsed": {
+        "message": "Last used",
+        "description": "Database list column title."
+    },
+    "optionsDatabaseCreated": {
+        "message": "Created",
+        "description": "Database list column title."
+    },
+    "optionsDatabaseDelete": {
+        "message": "Delete",
+        "description": "Database list column title."
+    },
+    "optionsColumnPageURL": {
+        "message": "Page URL",
+        "description": "Database list column title."
+    },
+    "optionsColumnDelete": {
+        "message": "Delete",
+        "description": "Database list column title."
+    },
+    "optionsSpecifiedFieldsNotFound": {
+        "message": "No specified credential fields found.",
+        "description": "Shown when no specified credentials are saved."
+    },
+    "optionsSpecifiedFieldsTabHelpTextFirst": {
+        "message": "If KeePassXC-Browser detects the wrong credential fields, you are able to specify the correct fields by yourself.",
+        "description": "Specified credential fields info text, first part."
+    },
+    "optionsSpecifiedFieldsTabHelpTextSecond": {
+        "message": "Just go to the page and click on the KeePassXC-Browser-Icon, now select ",
+        "description": "Specified credential fields info text, second part."
+    },
+    "optionsSpecifiedFieldsTabHelpTextThird": {
+        "message": "On this page you can manage theses specified credential fields.",
+        "description": "Specified credential fields info text, third part."
+    },
+    "optionsSpecifiedFieldsRemove": {
+        "message": "Remove specified credential fields?",
+        "description": "Confirmation text when removing specified credential fields."
+    },
+    "optionsSpecifiedFieldsConfirmation": {
+        "message": "Do you really want to remove the specified credential fields on the page:",
```

Comment:
I think I would generally replace "specified" with "custom" if that's what's meant.